src/AaveV3ListGhoStewards_20230507/GhoAaveV3GhoSteward_20230507_Payload.t.sol

```diff
@@ -0,0 +1,136 @@
+
+contract GhoAaveV3GhoSteward_20230507_Payload_Test is ProtocolV3TestBase {
+  GhoAaveV3GhoSteward public proposalPayload;
+  address immutable ZERO_ADDRESS = 0x0000000000000000000000000000000000000000;
+
+  function setUp() public {
+    vm.createSelectFork('https://rpc.tenderly.co/fork/4aa4b542-16b5-4fb7-8f75-fc1a0e2e3848');
+    proposalPayload = new GhoAaveV3GhoSteward();
+  }
+
+  function testGhoStewardRoles() public {
+    bool beforeGhoStewardAdmin = AaveV3Ethereum.ACL_MANAGER.isPoolAdmin(
+      proposalPayload.GHO_STEWARD()
+    );
+    assertFalse(beforeGhoStewardAdmin);
+
+    IGhoToken iGhoToken = IGhoToken(proposalPayload.GHO_TOKEN());
+
+    bool beforeHasBucketManagerRole = iGhoToken.hasRole(
+      iGhoToken.BUCKET_MANAGER_ROLE(),
+      proposalPayload.GHO_STEWARD()
+    );
+
+    assertFalse(beforeHasBucketManagerRole);
+
+    GovHelpers.executePayload(vm, address(proposalPayload), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    bool afterGhoStewardAdmin = AaveV3Ethereum.ACL_MANAGER.isPoolAdmin(
+      proposalPayload.GHO_STEWARD()
+    );
+    assertTrue(afterGhoStewardAdmin);
+
+    bool afterHasBucketManagerRole = iGhoToken.hasRole(
+      iGhoToken.BUCKET_MANAGER_ROLE(),
+      proposalPayload.GHO_STEWARD()
+    );
+
+    assertTrue(afterHasBucketManagerRole);
+  }
+
+  function testUpdateBorrowRates() public {
+    vm.startPrank(proposalPayload.RISK_COUNCIL());
+    GovHelpers.executePayload(vm, address(proposalPayload), AaveGovernanceV2.SHORT_EXECUTOR);
+
+    IPoolDataProvider iPoolDataProvider = IPoolDataProvider(
+      AaveV3Ethereum.AAVE_PROTOCOL_DATA_PROVIDER
+    );
+
+    IGhoSteward ghoSteward = IGhoSteward(proposalPayload.GHO_STEWARD());
+
+    address oldInterestStrategy = iPoolDataProvider.getInterestRateStrategyAddress(
+      proposalPayload.GHO_TOKEN()
+    );
+
+    uint256 oldBorrowRate = GhoInterestRateStrategy(oldInterestStrategy)
+      .getBaseVariableBorrowRate();
+
+    uint256 newBorrowRate = oldBorrowRate + 15;
+
+    ghoSteward.updateBorrowRate(newBorrowRate);
+
+    address newInterestStrategy = iPoolDataProvider.getInterestRateStrategyAddress(
+      proposalPayload.GHO_TOKEN()
+    );
+    assertEq(
+      GhoInterestRateStrategy(newInterestStrategy).getBaseVariableBorrowRate(),
+      newBorrowRate
+    );
+    vm.stopPrank();
+  }
+
+  function testUpdateBorrowRatesWhenNotRiskCouncil() public {
+    vm.startPrank(ZERO_ADDRESS);
+    GovHelpers.executePayload(vm, address(proposalPayload), AaveGovernanceV2.SHORT_EXECUTOR);
+    IGhoSteward ghoSteward = IGhoSteward(proposalPayload.GHO_STEWARD());
+    vm.expectRevert('INVALID_CALLER');
+
+    ghoSteward.updateBorrowRate(1);
+    vm.stopPrank();
+  }
+
+  function testUpdateBucketCapacity() public {
+    vm.startPrank(proposalPayload.RISK_COUNCIL());
+
+    IPoolDataProvider iPoolDataProvider = IPoolDataProvider(
+      AaveV3Ethereum.AAVE_PROTOCOL_DATA_PROVIDER
+    );
+
+    (address GHO_ATOKEN, , ) = iPoolDataProvider.getReserveTokensAddresses(
```

Comment:
use lower case for `ghoAToken`

Reply:
because its not a constant set in the contract?

Reply:
resolved 3f169bdfd649faeca2ddbfbe92d00350ee5be7c5